orgparse/node.py

```diff
@@ -10,6 +10,18 @@
 from .utils.py3compat import PY3, unicode
 
 
+def parse_lines(lines, env):
```

Comment:
any reason to move it to the top of file? IMO generally better to keep the code as intact as possible to make `git blame` easier

Reply:
I moved it there while diving into the architecture. Keeping changes minimal is a valid point though, i'll change it.